buffer/src/main/java/io/netty/buffer/UnpooledByteBufAllocator.java

```diff
@@ -24,7 +24,7 @@
 /**
  * Simplistic {@link ByteBufAllocator} implementation that does not pool anything.
  */
-public final class UnpooledByteBufAllocator extends AbstractByteBufAllocator implements ByteBufAllocatorMetricProvider {
+public class UnpooledByteBufAllocator extends AbstractByteBufAllocator implements ByteBufAllocatorMetricProvider {
```

Comment:
why we need this change ? Can we not just extend `AbstractByteBufAllocator` ?

Reply:
I am sure we can, I'm not too used to fiddling in this code base. Let me take a deeper look.